src/d3d10/meson.build

```diff
@@ -25,3 +24,10 @@ d3d10_core_dll = shared_library('d3d10core'+dll_ext, d3d10_core_src, d3d10_core_
 d3d10_core_dep = declare_dependency(
   link_with           : [ d3d10_core_dll ],
 )
+
+if platform != 'windows'
+  pkg.generate(d3d10_core_dll,
+    filebase: dxvk_pkg_prefix + 'd3d10core',
+    subdirs:  'dxvk',
```

Comment:
This must be consistent with the subdirectory we install into in `include/native/meson.build`, but is otherwise arbitrary.